libs/agno/tests/integration/models/aws/bedrock/test_async_basic.py

```diff
@@ -0,0 +1,161 @@
+import asyncio
+import pytest
+from pydantic import BaseModel, Field
+
+from agno.agent import Agent, RunResponse  # noqa
+from agno.models.aws import AwsBedrock
+from agno.storage.sqlite import SqliteStorage
+
+
```

Comment:
these async tests could be inside the `tests/integration/models/aws/test_basic.py` file itself.